dbt/README.md

```diff
@@ -62,6 +62,23 @@ To see all options for this command run:
 dbt_helper update-tables --help
 ```
 
+#### `summarize-row-count-diffs`
+When integrating new data, we often have to update row counts for many tables at once.
+In these cases, `dbt` does not provide any easy way to quickly see all failures from
+the `check_row_counts_per_partition` test, so we've added this command to grab all
+the failures from the most recent run of `dbt` tests, and summarize the results.
+
+The basic usage of this command is:
+
+```bash
+dbt_helper summarize-row-count-diffs
```

Comment:
I would add the `--verbose` flag because I can't really imagine using this helper function without it.